### PR TITLE
9.0 stock move partner info new

### PR DESCRIPTION
**Pull request by serpentcs-dev1**
## Stock Move Partner Info

- This module displays the partner of the picking in its stock movements and also allows grouping by a partner in stock movements.

### CODE REVIEW

File: stock_move_partner_info/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# Copyright 2015 Tecnativa - Pedro M. Baeza
+# Copyright 2015 AvanzOsc (http://www.avanzosc.es)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Stock Move Partner Info",
+    "version": "9.0.1.0.0",
+    "author": "OdooMRP team,"
+              "AvanzOSC,"
+              "Tecnativa - Pedro M. Baeza,"
```
Comment (pedrobaeza) [inline stock_move_partner_info/__openerp__.py]: Please only `Tecnativa`, and you can remove `OdooMRP team`, as that one is only temporary.

**Review by LoisRForgeFlow — APPROVED** [review]
I move my approval here :+1:

**Review by AaronHForgeFlow — APPROVED** [review]
Tested on runbot :+1:

**Comment by pedrobaeza** [comment]
I see that commit 3f5a181e18518e5df4cc562e4d3333591b0fd219, that contains main module still belongs to you, @darshan-serpent, why?

**Comment by AaronHForgeFlow** [comment]
It seems to me that the original author is not in git anymore

**Comment by serpentcs-dev1** [comment]
That commit belongs to @alfredo

**Comment by pedrobaeza** [comment]
Uhm, OK, @alfredoavanzosc do you want that the commit will be amended or do you agree to continue as is?

**Comment by alfredoavanzosc** [comment]
@pedrobaeza .. I agree to continue. :+1:

**Review by MiquelRForgeFlow — APPROVED** [review]
:+1:

File: stock_move_partner_info/models/stock_move.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+from openerp import fields, models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    picking_partner_id = fields.Many2one(
+        'res.partner', string='Picking Partner', store=True,
+        related='picking_id.partner_id', help='Partner of the picking',
+        readonly=True, oldname='picking_partner')
```
Comment (lmignon) [inline stock_move_partner_info/models/stock_move.py]: @darshan-serpent  a Nice To Have `index=True`

**Review by lmignon — APPROVED** [review]

**Comment by MiquelRForgeFlow** [comment]
Please @darshan-serpent squash last commit or reword it or @lmignon will receive a message each time anyone makes a fork that contains the commit

**Comment by serpentcs-dev1** [comment]
Really it's like that, I wasn't aware, thanks, @mreficent anyways I have squashed the commits.

All checks are green and the commits attribution are preserved too :)

@pedrobaeza I think this can be merged, right?

**Pull request merged**